lib/libdrm/BUILD

```diff
@@ -1,18 +1,12 @@
-mkdir build &&
-
-cd build &&
-
-meson --prefix /usr \
-      -Dudev=true \
-      -Damdgpu=auto \
-      -Dfreedreno=auto \
-      -Dexynos=auto \
-      -Domap=auto \
-      -Dtegra=auto \
-      $OPTS && 
-
-ninja &&
-
-prepare_install &&
-
-ninja install
+OPTS+=" --prefix /usr \
```

Comment:
Prefix already set by default_meson_build, also --prefix is not compatible with meson.